hoodie-cli/src/main/java/com/uber/hoodie/cli/commands/FileSystemViewCommand.java

```diff
@@ -0,0 +1,259 @@
+      @CliOption(key = {
+          "includeInflight"}, help = "Include Inflight Instants", unspecifiedDefaultValue = "false")
+          boolean includeInflight,
+      @CliOption(key = {"excludeCompaction"}, help = "Exclude compaction Instants", unspecifiedDefaultValue = "false")
+          boolean excludeCompaction,
+      @CliOption(key = {"limit"}, help = "Limit commits", unspecifiedDefaultValue = "-1") final Integer limit,
+      @CliOption(key = {"sortBy"}, help = "Sorting Field", unspecifiedDefaultValue = "") final String sortByField,
+      @CliOption(key = {"desc"}, help = "Ordering", unspecifiedDefaultValue = "false") final boolean descending,
+      @CliOption(key = {
+          "headeronly"}, help = "Print Header Only", unspecifiedDefaultValue = "false") final boolean headerOnly)
+      throws IOException {
+
+    HoodieTableFileSystemView fsView = buildFileSystemView(globRegex, maxInstant, readOptimizedOnly, includeMaxInstant,
+        includeInflight, excludeCompaction);
+    List<Comparable[]> rows = new ArrayList<>();
+    fsView.getAllFileGroups().forEach(fg -> fg.getAllFileSlices().forEach(fs -> {
+      int idx = 0;
+      Comparable[] row = new Comparable[readOptimizedOnly ? 5 : 8];
+      row[idx++] = fg.getPartitionPath();
+      row[idx++] = fg.getId();
+      row[idx++] = fs.getBaseInstantTime();
+      row[idx++] = fs.getDataFile().isPresent() ? fs.getDataFile().get().getPath() : "";
+      row[idx++] = fs.getDataFile().isPresent() ? fs.getDataFile().get().getFileSize() : -1;
+      if (!readOptimizedOnly) {
+        row[idx++] = fs.getLogFiles().count();
+        row[idx++] = fs.getLogFiles().filter(lf -> lf.getFileSize().isPresent())
+            .mapToLong(lf -> lf.getFileSize().get()).sum();
+        row[idx++] = fs.getLogFiles().filter(lf -> lf.getFileSize().isPresent())
+            .collect(Collectors.toList()).toString();
+      }
+      rows.add(row);
+    }));
+    Function<Object, String> converterFunction =
+        entry -> NumericUtils.humanReadableByteCount((Double.valueOf(entry.toString())));
+    Map<String, Function<Object, String>> fieldNameToConverterMap = new HashMap<>();
+    fieldNameToConverterMap.put("Total Delta File Size", converterFunction);
+    fieldNameToConverterMap.put("Data-File Size", converterFunction);
+
+    TableHeader header = new TableHeader()
+        .addTableHeaderField("Partition")
+        .addTableHeaderField("FileId")
+        .addTableHeaderField("Base-Instant")
+        .addTableHeaderField("Data-File")
+        .addTableHeaderField("Data-File Size");
+    if (!readOptimizedOnly) {
+      header = header.addTableHeaderField("Num Delta Files")
+          .addTableHeaderField("Total Delta File Size")
+          .addTableHeaderField("Delta Files");
+    }
+    return HoodiePrintHelper.print(header, fieldNameToConverterMap, sortByField, descending, limit, headerOnly, rows);
+  }
+
+  @CliCommand(value = "show fsview latest", help = "Show latest file-system view")
```

Comment:
In general, this command is very useful, but it's pretty involved as well, requiring deep knowledge of hoodie. I wonder if there's a way to simplify naming conventions for anybody to understand and use this command

Reply:
May be add some comments before each operation to say what it's result is supposed to mean ?